include/spdlog/sinks/ansicolor_sink.h

```diff
@@ -33,6 +33,40 @@ class ansicolor_sink : public sink
     virtual void log(const details::log_msg& msg) override;
     virtual void flush() override;
 
+#ifdef _WIN32
+    void set_color(level::level_enum level, const short& color);
+
+	/// Formatting codes
+    const short reset      = 0;
+    const short bold       = FOREGROUND_INTENSITY;
+    const short dark       = reset; // Not implemented in windows

```

Comment:
Can you use ANSI sequences on Windows automatically if `ANSICON` environment variable is set, because windows console not support all features. Programs like ConEmu or AnsiCon set this variable when they support ANSI escape codes (http://conemu.github.io/en/AnsiEscapeCodes.html).